fast_curator/catalogues/__init__.py

```diff
@@ -53,6 +53,12 @@ def check_entries_uproot(files, tree_names, no_empty, confirm_tree=True, list_br
     if not isinstance(tree_names, (tuple, list)):
         tree_names = [tree_names]
 
+    if len(files) > 0:
```

Comment:
Thanks for this Sam.  I'm not sure this should be the default behaviour though.  It ought to be controlled by an option like `ignore_inacessible` trees.  I think it's dangerous if we just silently handle bad input files, since people might not be aware that the files are bad and need them to be processed, which result in an unexpectedly low number of events being processed. 

The question maybe is what are the right sort of flags and controls to provide and how should things be reported.

Reply:
it's somewhat related to confirm_tree/no_empty -> ie doesn't handle
or do you think the best approach is just a new option?

Related to this, do you think it's worth having an option to output the files which were ignored so people know?